users/api.py

```diff
@@ -142,3 +144,38 @@ def fetch_users(filter_values, ignore_case=True):
             )
         )
     return user_qset
+
+
+def find_available_username(initial_username_base):
```

Comment:
Could you add some code comments to this function annotating what is going on (i.e. incrementing the suffix and truncating the username to be sure it fits in the username field)?

Reply:
Meant to do that, completely forgot